lib/Site.js

```diff
@@ -259,6 +259,7 @@ Site.prototype.createPage = function (config) {
     sourcePath,
     tempPath,
     resultPath,
+    searchable: config.searchable,
```

Comment:
Let's move this to below line 254 to maintain alphabetical order (only the `sourcePath`, `tempPath`, `resultPath` and `asset` are placed at the bottom).